tests/s3/test_bucket_tagging.py

```diff
@@ -170,7 +171,8 @@ def test_2434(self):
     def test_2435(self):
         """Create a tag whose key is up to 128 Unicode characters in length."""
         self.log.info(
-            "STARTED: Create a tag whose key is up to 128 Unicode characters in length")
+            "STARTED: Create a tag whose key is up to 128 Unicode "
```

Comment:
Why a different style is followed for log messages. It seems different from rest of the comments.